app/upgrades/v20/upgrades_test.go

```diff
@@ -142,3 +145,163 @@ func TestUpdateExpeditedPropsParams(t *testing.T) {
 		})
 	}
 }
+
+var metaDatas = []banktypes.Metadata{
+	{
+		Description: "The native EVM, governance and staking token of the Evmos Hub",
+		DenomUnits: []*banktypes.DenomUnit{
+			{
+				Denom:   "aevmos",
+				Aliases: []string{"attoevmos"},
+			},
+			{
+				Denom:    "evmos",
+				Exponent: 18,
+			},
+		},
+		Base:    "aevmos",
+		Display: "evmos",
+		Name:    "Evmos",
+		Symbol:  "EVMOS",
+	},
+	{
+		Description: "Cosmos coin token representation of 0x153A59d48AcEAbedbDCf7a13F67Ae52b434B810B",
+		DenomUnits: []*banktypes.DenomUnit{
+			{
+				Denom: "erc20/0x153A59d48AcEAbedbDCf7a13F67Ae52b434B810B",
+			},
+			{
+				Denom:    "WrappedEtherCeler",
+				Exponent: 18,
+			},
+		},
+		Base:    "erc20/0x153A59d48AcEAbedbDCf7a13F67Ae52b434B810B",
+		Display: "WrappedEtherCeler",
+		Name:    "erc20/0x153A59d48AcEAbedbDCf7a13F67Ae52b434B810B",
+		Symbol:  "ceWETH",
+	},
+	{
+		Description: "Staking derivative stATOM for staked ATOM by Stride",
+		DenomUnits: []*banktypes.DenomUnit{
+			{
+				Denom:   "ibc/0830AFFC2F4F7CD24F9CEC07024FEA64CE3C5ABBC520DBD803BFA97BC3DCCA85",
+				Aliases: []string{"stuatom"},
+			},
+			{
+				Denom:    "statom",
+				Exponent: 6,
+			},
+		},
+		Base:    "ibc/0830AFFC2F4F7CD24F9CEC07024FEA64CE3C5ABBC520DBD803BFA97BC3DCCA85",
+		Display: "statom",
+		Name:    "Stride Staked Atom",
+		Symbol:  "stATOM",
+	},
+}
+
+func TestFixDenomMetadata(t *testing.T) {
+	var (
+		nw  *network.UnitTestNetwork
+		ctx sdk.Context
+	)
+	testCases := []struct {
```

Comment:
Can we also add a check on expected result after migration? 🙏

Reply:
yeah, it is the same for all. It checks that the denoms are stored properly